lib/workers/repository/process/lookup/index.spec.ts

```diff
@@ -341,6 +342,19 @@ describe('workers/repository/process/lookup/index', () => {
       expect(res.updates).toMatchSnapshot();
       expect(res.updates[0].updateType).toBe('minor');
     });
+
+    it('handles the in-range-only strategy', async () => {
+      config.currentValue = '~1.2.0';
+      config.lockedVersion = '1.2.0';
+      config.rangeStrategy = 'in-range-only';
+      config.depName = 'q';
+      config.datasource = datasourceNpmId;
+      httpMock.scope('https://registry.npmjs.org').get('/q').reply(200, qJson);
+      const res = await lookup.lookupUpdates(config);
+      expect(res.updates).toMatchSnapshot();
+      expect(res.updates[0].updateType).toBe('minor');
```

Comment:
Line 355 has a `TypeError: Cannot read property 'updateType' of undefined` problem.